app/services/orders/bulk_cancel_service.rb

```diff
@@ -10,11 +10,14 @@ def initialize(params, current_user)
     end
 
     def call
-      editable_orders.where(id: @order_ids).each do |order|
+      cancelled_orders = []
+      editable_orders.where(id: @order_ids).find_each do |order|
         order.send_cancellation_email = @send_cancellation_email
         order.restock_items = @restock_items
         order.cancel
+        cancelled_orders << order
       end
+      cancelled_orders
```

Comment:
It took me a few minutes to work out why this was needed: because `each` returns the array of records, but `find_each` doesn't retain an array, to avoid taking up too much memory.

Hmm, so then if we need to store the records in an array, there's no reason to use `find_each` here!

`±` Could you please change this one back to `each` with a `rubocop:disable`?

Reply:
Theres some (little) benefit in retaining the `find_each` since the processing will be done in batches.  The `cancelled_orders` array will expand gradually rather than suddenly.

The little benefit is negated by the lack of readability.

Will use a `rubocop:disable` for this one.